dace/data.py

```diff
@@ -253,6 +259,11 @@ class Array(Data):
                          desc='This pointer may alias with other pointers in '
                          'the same function')
 
+    alignment = Property(dtype=int,
```

Comment:
This doesn't actually change anything for I/O arrays, right? In those cases, we should sanity check than I/O arrays are aligned to what is expected, otherwise we could end up with horrible sporadic runtime errors.

Reply:
Correct. If referring to non-transient memory, alignment (same as the strides) has to match the external memory attributes. We cannot really verify this until SDFG call-site, and we try to make that as low overhead as possible.

Reply:
It's a very cheap check though: `ptr % alignment == 0`, and it only emitted if alignment is actually set. Seems worth it?

Reply:
Sure, added to TODO

Reply:
For this PR or TODO(later)? Kappa

Reply:
As part of the new code generator Project. So later, but not that much later (once we actually use these properties to generate code).